development/image-syncer/external-images.yaml

```
@@ -45,8 +45,8 @@ images:
 - source: "quay.io/prometheus-operator/prometheus-operator:v0.61.1"
 - source: "quay.io/prometheus/alertmanager:v0.24.0"
 - source: "quay.io/prometheus/prometheus:v2.40.7"
-- source: "nats@sha256:14020946e383f9af1e62d3df38f3192e1a2f6619cec7e224eff54e997fc96d77"
-  tag: "2.9.11-alpine3.17"
+- source: "nats@sha256:2a0dd7504f75410d3b3198dbe094b70521260161f86ce9ec6a90b87548e5a22e"
```

We're now using Image index SHA to sync all architectures
```suggestion
- source: "nats@sha256:23353a80fc1fecc883623f6b72759544f596be792aed1db01257ac900ca8d47f"
```